src/app/recipe/new/_components/IngredientListItem.tsx

```diff
@@ -0,0 +1,14 @@
+import Link from "next/link";
+
+type Props = {
+  title: string;
+  external?: boolean;
```

Comment:
基本的にboolで判断するような値はisとかhasを付けましょう！
多分boolの命名は下みたいにするのが共通認識なエンジニアが多数です。（違ったらすみません）
https://qiita.com/GinGinDako/items/6e8b696c4734b8e92d2b

またexternalだと他の人が見た時に何のためのpropなのかわかりにくいので多少長くなっても分かりやすい名前か[こんな感じで](https://github.com/qin-team-recipe/10-recipe-app/blob/abdd002aad0f540ef1278b6f60c8bc84d090a47d/src/components/image/ImageComponent.tsx#L6-L7)コメント残してあげてください。

```suggestion
  isExternalLink?: boolean;
```